### ISSUE TITLE
Conceptually change virtual points system to 'impact effect' on real world for long-term motivation

### ISSUE THREAD

**Issue by CaptSpot** [veganaut/veganaut#163]
While the points system of Veganaut is easy to understand, there's no real *emotional* connection to them - or at least they're out of a human context/ don't bridge to the physical world.

The questions remains: Why should one collect points in the first place? To compare to others? And then to brag about them when you're high up in the high-score list? Can this keep one motivated enough in the long term? I don't think so.

While visiting a few vegan restaurants in San Francisco I discovered a small poster that showed how many gallons of water, pounds of grain, square feet of rain forest, gallons of gasoline etc. you can save if you 'go vegan' for 1 day a year long (source: http://wannaveg.com/). How about getting rid of the virtual points system and actually try to convert any action into an impact value to the real world?

As simple as:

 > I eat vegan 1 day...
 > = Equals n gallons of water saved, 1 pound of grain etc.

*Every* action should add to your score and communicate a positive impact on the real world. This also gives you more flexibility to evaluate one's actions in a more diverse way. In addition it allows further visualisations of the impact instead of using conceptual abstractions. This would keep me so much more motivated!

**Comment by SebuLeugger** [comment]
I agree with all the points you bring up here. The current point and ranking system was easy to implement but it doesn't connect to the real world. We want to change that. It's difficult/impossible to break every action down to a direct impact value, e.g. what's the impact value of asking for vegan options or of rating an existing vegan option? However, if you buy a vegan option there's a pretty straight forward way to translate this into an impact value: the average amount of resources saved by eating 1 vegan meal instead of an average omnivore meal. For all the other actions we have to find something else. 

I'll think about this more. Thanks for the nudge and for the suggestions!

**Comment by SebuLeugger** [comment]
What do you think of this idea: https://github.com/veganaut/veganaut/issues/32#issuecomment-108277857 ?

**Comment by CaptSpot** [comment]
I see, like +1 or 'Like' on other social media platforms. My fear is that it reflects 'yet another like functionality' and that the direct impact of the proposed action isn't clear to your audience. The simple question is: 'What's in for me?' and if this question can't be answered then 'What's in for others?' - the fact that you're going to mine this information and then prioritise certain information (?) may not be instantly clear. I'm worried that it's shifting the problem to another part of the system but not actually solving it. You have one clear advantage in comparison to other platforms: You're talking about food. Something real. And you have a precise focus (vegan food), which should allow you to do it better than a generic social media platform.

**Comment by SebuLeugger** [comment]
How about just telling people how many other people looked at the information they entered and make that the relevant stat? This wouldn't require other users to "like" or +1 anything, but it would still give users a feel for the impact they are having.

**Comment by CaptSpot** [comment]
Kind of a simple "frequency meter" idea that directly correlates with a positive impact on the environment, since the products being sold at a specific place are more sustainable than in 'regular' places? Yup, this has a certain potential.
